font/grub2font.ksy

```diff
@@ -103,3 +103,22 @@ types:
             type: u1
           - id: offset
             type: u4
+        instances:
+          bitmap:
+            io: _root._io
+            pos: offset
+            type: character_definition
+      character_definition:
+        seq:
+          - id: width
+            type: u2
+          - id: height
+            type: u2
+          - id: x_offset
+            type: u2
+          - id: y_offset
+            type: u2
+          - id: device_width
+            type: u2
+          - id: bitmap_data
+            size: (width * height + 7) / 8
```

Comment:
```suggestion
            size: (width * height + 7) / 8 # ceiled integer division
```

I'd add this comment that might save time and reduce confusion when stumbling across this magical calulcation and trying to figure out what you meant by this.

Also, this means that every pixel always takes up 1 bit, right? What is the color palette, then? Is it `0 = black` and `1 = white`, or `0 = primary color` (it's a font, so it's probably meant to be dyed an arbitrary color I suppose) and `1 = transparent` or vice versa? Write in the `doc` key, please.

In addition to the comment `# ceiled integer division`, it would make sense to mention in `doc` what this ceiling does in practice: as I understand it, `bitmap_data` is an actually a [bit array](https://en.wikipedia.org/wiki/Bit_array) of `width * height` bits whose end position is aligned to the closest next byte boundary. I think it's beneficial to acquaint the spec readers/users with this concept, because it's an integral trait of the format.

Reply:
The documentation doesn't say anything about colour. The source code says each pixel is 1 bit.

Reply:
Regarding 2 & 3: I wrote something, not sure if it makes sense to you.